src/4_initialization.js

```diff
@@ -23,12 +23,13 @@ if (window['branch'] && window['branch']['_q']) {
 
 // AMD
 if (typeof define === 'function' && define.amd) {
-	define('branch', function() { return branch_instance; });
+	define(['branch'], function() { return branch_instance; });
```

Comment:
very minor code style thing - i think the style is to put a space around the quoted string - not sure why the linter didn't catch it